ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-master-presubmits.yaml

```diff
@@ -10,7 +10,7 @@ presubmits:
       skip_cloning: true
     labels:
       ci-operator.openshift.io/prowgen-controlled: "true"
-    name: pull-ci-openshift-cluster-network-operator-master-e2e-aws
+    name: pull-ci-openshift-cluster-network-operator-master-e2e-aws-openshift-sdn
```

Comment:
If we're renaming this we should rename it consistently (eg, change the `rerun_command` too). I'm not sure whether it's better to rename it (and be clear) or not rename it (and be consistent with everyone else). Maybe slight preference toward rename...